Mathlib/Data/Finset/Sups.lean

```diff
@@ -468,17 +468,16 @@ theorem disjSups_inter_subset_left : (s₁ ∩ s₂) ○ t ⊆ s₁ ○ t ∩ s
 theorem disjSups_inter_subset_right : s ○ (t₁ ∩ t₂) ⊆ s ○ t₁ ∩ s ○ t₂ := by
   simpa only [disjSups, product_inter, filter_inter_distrib] using image_inter_subset _ _ _
 
-variable (s t)
-
-theorem disjSups_comm : s ○ t = t ○ s := by
```

Comment:
Shouldn't we also keep this version of the statement? Idem for the `assoc` version below

Reply:
All right, the original theorems have been restored.